docs/source/app-dev/bindings-js/index.rst

```diff
@@ -7,16 +7,50 @@ Node.js bindings
 .. toctree::
   :hidden:
 
-  getting-started
   tutorial
   daml-as-json
 
 The Node.js bindings let you access the Ledger API from Node.js code.
 
-The Installing page guides you through the process of :doc:`getting-started`.
+Getting started
+***************
 
-After that, a tutorial teaches you :doc:`how to build a simple application from scratch <tutorial>`.
+1. Set up NPM
+=============
 
-Finally, `the API reference documentation`_ and :doc:`an introduction to DAML as JSON <daml-as-json>` are also available.
+The Node.js bindings are published on NPM.
```

Comment:
I appreciate your optimism. :smile:

Reply:
After reading the following lines, I realize you are probably not talking about the public NPM. Without qualification, "published on NPM" would lead people to think it's published on the public one (which is the plan but still has to happen).